analysed-packages/libmnl/version-1.0.4/libmnl-1.0.4-SPDX2TV.spdx

```diff
@@ -0,0 +1,2997 @@
+LicenseInfoInFile: LicenseRef-fsf-free
+LicenseInfoInFile: LicenseRef-libtool-exception-2.0
+FileCopyrightText: <text> Copyright (C) 1992-1996, 1998-2012 Free Software Foundation, Inc.
+Copyright (C) 2012 Free Software Foundation, Inc.
+Copyright (C) 2012 Free Software Foundation, Inc.
+Copyright (C) 1996, 1997, 1998, 1999, 2000, 2001, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011 Free Software Foundation, Inc. Written by Gordon Matzigkeit, 1996 </text>
+
+##File
+
+FileName: libmnl-1.0.4.tar.bz2/libmnl-1.0.4.tar/libmnl-1.0.4/m4/libtool.m4
+SPDXID: SPDXRef-item461316
+FileChecksum: SHA1: df1b378e235c9f5df79c68c8fbae6e47a1d219c9
+FileChecksum: SHA256: b9b328e874d9278ec27f2b47f6ed11c1f2eca4a9cb9ebf945e990f65324db97b
+FileChecksum: MD5: 42a15e936131c8cbcc3c88b839022f8a
+LicenseConcluded: LicenseRef-FSFULLR AND LicenseRef-GPL-2.0-or-later-WITH-Libtool-exception
+LicenseComments: <text> The information in the file is:
+
+This file is part of GNU Libtool.
+
+GNU Libtool is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License as
+published by the Free Software Foundation; either version 2 of
+the License, or (at your option) any later version.
+
+As a special exception to the GNU General Public License,
+if you distribute this file as part of a program or library that
+is built using GNU Libtool, you may include this file under the
+same distribution terms that you use for the rest of that program.
+
+GNU Libtool is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+Thus, we concluded GPL-2.0-or-later WITH Libtool-exception. </text>
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Libtool-exception
+LicenseInfoInFile: LicenseRef-FSFUL
+LicenseInfoInFile: LicenseRef-FSFULLR
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+LicenseInfoInFile: LicenseRef-fsf-unlimited
+LicenseInfoInFile: LicenseRef-fsf-free
+LicenseInfoInFile: LicenseRef-libtool-exception-2.0
+FileCopyrightText: <text> Copyright (C) 1996, 1997, 1998, 1999, 2000, 2001, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011 Free Software Foundation, Inc. Written by Gordon Matzigkeit, 1996
+Copyright (C) 1996, 1997, 1998, 1999, 2000, 2001, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011 Free Software Foundation, Inc. Written by Gordon Matzigkeit, 1996
+Copyright (C) 2011 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: libmnl-1.0.4.tar.bz2/libmnl-1.0.4.tar/libmnl-1.0.4/aclocal.m4
+SPDXID: SPDXRef-item461314
+FileChecksum: SHA1: 42181c290d6d90039f46a8e88a34ed512893c843
+FileChecksum: SHA256: 5aaa29084b0c6a7e556a325790fb5b75045f0c08116898ba1f19b3a0c7dcb1d9
+FileChecksum: MD5: 9a5304ae05de4fe7476c7473adb64546
+LicenseConcluded: LicenseRef-FSFULLR AND LicenseRef-FSFULLRWD
```

Comment:
The conclusion of LicenseRef-FSFULLRWD is correct but the text would be here:
This file is free software; the Free Software Foundation
gives unlimited permission to copy and/or distribute it,
with or without modifications, as long as this notice is preserved.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY, to the extent permitted by law; without
even the implied warranty of MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.
And not 

> This Makefile.in is free software; ...